### ISSUE TITLE
Ground-Truth information

### ISSUE THREAD

**Issue by zouaghi96** [jiaan40504/rPPG#2]
Hi Sir,

I have a question concerning the ground-truth information used in the dataset. Is it the PPG signal values or the heart rate values?
For the training, only the heart rate ground-truth is used? The signal is also fed to the model as an input or not?

does the Pearson correlation used for computing the loss function aim to compute the correlation between the ground-truth and the output which is the HR?

the Siamese with HR estimator computes only the HR, and not the signal? Right?
Is there any loss computation between the output of conv123456 and the ground-truth? if yes what is the ground-truth used for this computation?

Thank you in advance.
